From: Jan Rybar <jrybar@redhat.com>
Date: Wed, 15 Aug 2018 18:56:43 +0200
Subject: Error message raised on every 'systemctl start' in emergency.target

Superuser should know that polkit is not running in emergency.target.
If not, basic info with debug sources is offered instead of error message.
Other usecases taken into account.

Origin: upstream, 0.116, commit:8c1bc8ab182f33a55503d30aa7a4ee96f822d903
---
 src/programs/pkttyagent.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/programs/pkttyagent.c b/src/programs/pkttyagent.c
index 488ca8b..fe74765 100644
--- a/src/programs/pkttyagent.c
+++ b/src/programs/pkttyagent.c
@@ -180,7 +180,8 @@ main (int argc, char *argv[])
   authority = polkit_authority_get_sync (NULL /* GCancellable* */, &error);
   if (authority == NULL)
     {
-      g_printerr ("Error getting authority: %s (%s, %d)\n",
+      g_printerr ("Authorization not available. Check if polkit service is running or see debug message for more information.\n");
+      g_debug ("Error getting authority: %s (%s, %d)\n",
                   error->message, g_quark_to_string (error->domain), error->code);
       g_error_free (error);
       ret = 127;
